kof-operator/internal/controller/isito/remote_secret.go

```diff
@@ -259,6 +259,10 @@ func getServerFromKubeconfig(client kube.CLIClient) (string, multicluster.Warnin
 	return server, nil, nil
 }
 
+func CopyRemoteSecretProfileName(childClusterName string) string {
+	return fmt.Sprintf("%s-istio-remote-secret", childClusterName)
```

Comment:
`childClusterName` has type `string`, so let's keep it simple:
```suggestion
	return childClusterName+"-istio-remote-secret"
```